tests/layer_tests/pytorch_tests/test_roll.py

```diff
@@ -18,12 +18,12 @@ class aten_roll(torch.nn.Module):
             def __init__(self, shifts, dim=None):
                 super(aten_roll, self).__init__()
                 self.dim = dim
-                self.shits = shifts
+                self.shifts = shifts
```

Comment:
Could you also fix same problem here:
https://github.com/openvinotoolkit/openvino/blob/221847de5b24fcc47d82c8774f8f8a8c09d2df51/tests/layer_tests/py_frontend_tests/test_torch_decoder.py#L482